interactive_templates/templates/v2/analysis/measures.py

```diff
@@ -2,7 +2,7 @@
 from pathlib import Path
 
 import pandas as pd
-from analysis.report_utils import get_date_input_file, match_input_files
+from analysis.report_utils import calculate_rate, get_date_input_file, match_input_files
 
 
 def round_column(df, col, decimals=-1):
```

Comment:
Could rename redact_and_round or round_and_redact, and the name "round" does not indicate that it also does redaction